warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexRegexIteratorJexl.java

```diff
@@ -93,6 +95,10 @@ public String toString() {
             startKey = new Key(rowId, fiName);
             endKey = new Key(rowId, new Text(fiName.toString() + '\0'));
             return new RangeSplitter(new Range(startKey, true, endKey, true), getMaxRangeSplit());
+        } else if (ANY_FINAME.equals(fiName)) {
```

Comment:
what happens if we have a negated ANY_FINAME

Reply:
You are correct.  Moving anyfield logic up